lib/SILOptimizer/Utils/SILInliner.cpp

```diff
@@ -179,6 +188,26 @@ void SILInliner::inlineFunction(FullApplySite AI, ArrayRef<SILValue> Args) {
   }
 }
 
+SILValue SILInliner::borrowFunctionArgument(SILValue callArg,
+                                            FullApplySite AI) {
+  if (!AI.getFunction()->hasQualifiedOwnership()
+      || callArg.getOwnershipKind() != ValueOwnershipKind::Owned) {
+    return callArg;
+  }
+  auto *borrow = getBuilder().createBeginBorrow(AI.getLoc(), callArg);
+  if (auto tryAI = dyn_cast<TryApplyInst>(AI)) {
```

Comment:
Can you put in the * here? We have been pretty consistent about that in the optimizer.

Reply:
Thanks. That's a copy-pasto.